compact/optical_materials.xml

```diff
@@ -258,6 +258,39 @@
       4.75035*eV   1.85713
       4.93961*eV   1.87039
       "/>
+    <matrix name="RINDEX__VM2000" coldim="2" values="
+      1.37760*eV   1.42
+      1.45864*eV   1.42
+      1.54980*eV   1.42
+      1.65312*eV   1.42
+      1.71013*eV   1.42
+      1.77120*eV   1.42
+      1.83680*eV   1.42
+      1.90745*eV   1.42
+      1.98375*eV   1.42
+      2.06640*eV   1.42
+      2.10143*eV   1.42
+      2.13766*eV   1.42
+      2.17516*eV   1.42
+      2.21400*eV   1.42
+      2.25426*eV   1.42
+      2.29600*eV   1.42
+      2.33932*eV   1.42
+      2.38431*eV   1.42
+      2.43106*eV   1.42
+      2.47968*eV   1.42
+      2.53029*eV   1.42
+      2.58300*eV   1.42
+      2.63796*eV   1.42
+      2.69531*eV   1.42
+      2.75520*eV   1.42
+      2.81782*eV   1.42
+      2.88335*eV   1.42
+      2.95200*eV   1.42
+      3.09960*eV   1.42
+      3.54241*eV   1.42
+      4.13281*eV   1.42
+      "/>
```

Comment:
Why do we need all these values if they are the same?

Reply:
Here I simply copy paste those values from Fun4all. I could reduce all these into 2 values, like these,
1.37760*eV   1.42
4.13281*eV   1.42